src/main/java/uk/gov/companieshouse/overseasentitiesapi/validation/utils/CountryLists.java

```diff
@@ -0,0 +1,288 @@
+package uk.gov.companieshouse.overseasentitiesapi.validation.utils;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+public class CountryLists {
+
+        public static final List<String> overseasEntityCountryList;
```

Comment:
Cleaner to make these private and have getter methods to access them. I also wouldn't mention 'List' in the end of the variable and method names as that's implied by the class name (`CountryLists`). So `ukCountries` and `getUkCountries()` for example.

Reply:
Hidden these and using getter methods.